spec.html

```diff
@@ -11832,14 +11832,29 @@ <h1>Execution Contexts</h1>
             Generator
           </td>
           <td>
-            The Generator that this execution context is evaluating.
+            The Generator that is associated with this execution context.
           </td>
         </tr>
       </table>
     </emu-table>
-    <p>In most situations only the running execution context (the top of the execution context stack) is directly manipulated by algorithms within this specification. Hence when the terms “LexicalEnvironment”, and “VariableEnvironment” are used without qualification they are in reference to those components of the running execution context.</p>
+    <p>To express the semantics of generators, this document uses steps of the form “<dfn id="transfer-control" variants="transfers control,transfer of control">Transfer control</dfn> from _aContext_ to _bContext_, passing _completion_”, where _aContext_ and _bContext_ are execution contexts and _completion_ is a Completion Record. This means:</p>
+    <ul>
+      <li>Execution of the current algorithm (the one with the “Transfer control” step) is suspended, and a representation of that suspended execution is saved in the code evaluation state of _aContext_.</li>
+      <li>The code evaluation state of _bContext_ is typically a representation of an execution that was suspended in another “Transfer control” step; that execution is resumed, and _completion_ is provided as “the Completion Record passed when control was transferred back to _bContext_”. Alternatively, _bContext_'s code evaluation state is an Abstract Closure; in that case, the Abstract Closure is called with no arguments, and _completion_ is ignored.</li>
+    </ul>
+    <p>The suspended execution associated with _aContext_ will stay suspended until some step transfers control back to _aContext_, which might never happen.</p>
+    <emu-note>This is analogous to the concept of coroutines in programming languages.</emu-note>
+    <emu-note>In practice, _aContext_ was the running execution context until just before the “transfer control” step, and _bContext_ is the new running execution context. However, the “transfer control” step itself doesn't manipulate the execution context stack.</emu-note>
```

Comment:
Why is this note needed?

Reply:
😆 I just noticed that I asked this same thing in my original review: https://github.com/tc39/ecma262/pull/2962#discussion_r1086966494